player.py

```diff
@@ -0,0 +1,32 @@
+def percent_diff(a, b):
+    return ((a - b) / b) * 100
+
+
+def alert_open(results, last_trade):
+    a = results[len(results) - 1]['price']
+    b = results[len(results) - 2]['price']
+    c = results[len(results) - 3]['price']
+    if len(last_trade) == 0 or last_trade[0]['buy'] is False:
```

Comment:
consider using `and` instead of encapsulated `if` statements for better readability:
```py
if (len(last_trade) == 0 or last_trade[0]['buy'] is False) and percent_diff(b, c) < 0 and percent_diff(a, b) >= 0.1:
    return True    
```